share/functions/help.fish

```diff
@@ -141,6 +141,18 @@ function help --description 'Show help for the fish shell'
         if type -q cygpath
             set page_url file://(cygpath -m $__fish_help_dir)/$fish_help_page
         end
+
+        # For WSL, provide full path to the file
+        if string match -qr 'cmd.exe$' -- $fish_browser[1]
+            set requires_windows_path 1
+        else if string match -qr 'www-browser' -- $fish_browser[1]
+            if string match -qr 'wslview' (builtin realpath (which www-browser))
```

Comment:
prefer the builtin `command -s` over `which` /which/ may not be installed on some systems